01/Task01.js

```diff
@@ -1,16 +1,31 @@
 import React from 'react';
-
+import { ThemeProvider } from 'styled-components';
 import Alert from './../src/components/Alert';
 import { Row, Col, Alert as RBAlert } from 'react-bootstrap';
 
 const Task01 = () => {
+    const themeSettings = {
```

Comment:
Docelowo "theme" definiujemy w osobnym pliku i jest on przekazywany gdzieś na "górze" np. w `App.js`.
Na potrzeby zadań jest ok, ale w projekcie lepiej to zrobić tak jak piszę wyżej i zazwyczaj jest to jeden plik :)


PS. Wiem, że wygodniej jest trzymać całe deklaracje w motywie, ale lepiej tam trzymać tylko wartości tj. kolor, grubość obramowania itp. :)